docs/en/observability/uptime-set-up-heartbeat.asciidoc

```diff
@@ -0,0 +1,130 @@
+[[uptime-set-up-choose-heartbeat]]
+= Use {heartbeat}
+
+{heartbeat} is a lightweight daemon that you install on a remote server to periodically
+check the status of your services and determine if they are available. Performance data is
+then gathered and reported back to {heartbeat}, where it's formatted and sent to the {stack}.
+
+[discrete]
+[[uptime-set-up-docker]]
+== Pull the Docker image
+
+Elastic provides Docker images that you can use to run monitors, which reduces the chance of problems.
+Start by pulling the {heartbeat} Docker image.
+
+// ifeval::["{release-state}"=="unreleased"]
+
+// Version {version} has not yet been released.
+
+// endif::[]
+
+// ifeval::["{release-state}"!="unreleased"]
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/beats/heartbeat:{version}
+----
+
+// endif::[]
+
+[discrete]
+[[uptime-set-up-config]]
+== Configure
+
+Next, create a `heartbeat.yml` configuration file.
+
+There are four possible monitor types.
```

Comment:
Do we need to say there are four types if we're going to explain HTTP straightaway? IMO we should either list what the types are or (preferably IMO) remove this phrase.

It seems a bit out of context here.